### PR TITLE
Install flake8, black and isort for linting and reformatting

### LINKED ISSUE #69: Remove unused imports
VSCode likes to import the object when you mistype and choose a object in a context menu. When a developer removes the mistyped object, they often forget to remove the import at the top of the file.

### LINKED ISSUE #105: Install flake8 and black for linting/formatting
- [x] Install flake8 and black as dependencies
- [x] Remove autopep8
- [x] Resolve linting issues
- [x] Format the entire project using black
- [x] Update python-app.yml

### LINKED ISSUE #113: Install isort to sort import statements
- [x] Install isort using poetry
- [x] isort entire project

### PR DESCRIPTION
**Pull request by Exsects**
See #105 and #113 for reference.

### CODE REVIEW

**Review by Karan-Jain — DISMISSED** [review]

**Comment by Karan-Jain** [comment]
Looks like these are going to run with pipeline. Nice!

File: .vscode/launch.json [hunk 1]
```diff
@@ -1,24 +0,0 @@
-{
```
Comment (borisliao) [inline .vscode/launch.json]: Add back this file

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,8 +0,0 @@
-{
```
Comment (borisliao) [inline .vscode/settings.json]: Add back this file

**Review by borisliao — CHANGES_REQUESTED** [review]
Add back pytest.ini file, that might fix unit tests

**Review by borisliao — APPROVED** [review]
LGTM

**Review by Karan-Jain — APPROVED** [review]

**Pull request merged**